src/wasm/kotlin/java/arcs/RuntimeClient.kt

```diff
@@ -0,0 +1,86 @@
+package arcs
+
+/**
+ * Delegate core runtime operations to the appropriate platform.
+ */
+expect object RuntimeClient {
+    /**
+     * Remove the entity within the Singleton handle of [particle].
+     *
+     * @param particle Particle in scope
+     * @param singleton target to be cleared
+     */
+    fun <T : Entity<T>> singletonClear(particle: Particle, singleton: Singleton<T>)
+
+    /**
+     * Set the entity of the Singleton handle of [particle].
+     *
+     * @param particle Particle in scope
+     * @param singleton target to be cleared
+     * @param encoded Serialized representation of an entity
```

Comment:
Nit: @param descriptions start with a lowercase (so lowercase `serialized` here). Same for `particle`, throughout file